crates/stackable-operator/src/crd/listener/class/v1alpha1_impl.rs

```diff
@@ -0,0 +1,19 @@
+use crate::crd::listener::{
+    class::v1alpha1::ListenerClassSpec,
+    core::v1alpha1::{AddressType, KubernetesTrafficPolicy, PreferredAddressType},
+};
+
+impl ListenerClassSpec {
```

Comment:
Hm, is the expectation that we'll copy/paste this for every new version?

Reply:
This is a two edged-sword:

- Between some versions, there will be no changes to impl blocks (which would require pesky copy/paste)
- Between other versions, there _will_ be changes, so a dedicated impl block is needed.

There _might_ be something we can do for shared functionality using the macro, but I didn't think too hard about that and it also might be very tricky to do.

Reply:
True, we can always revisit eventually.